lib/pbench/test/unit/server/test_pbench_create_user.py

```diff
@@ -46,12 +47,15 @@ def test_help(pytestconfig):
 
     @staticmethod
     @responses.activate
-    def test_args(valid_config, pytestconfig):
+    def test_args(pytestconfig):
+        cfg_file = pytestconfig.cache.get("_PBENCH_SERVER_CONFIG", None)
```

Comment:
On the agent side, this is handled via the `valid_config` Pytest fixture...you don't want to set up something analogous for the server side and avoid the duplicate code?

Reply:
Fixed